packages/netlify-cms-widget-list/src/ListControl.js

```diff
@@ -411,6 +411,7 @@ export default class ListControl extends React.Component {
               ref={this.processControlRef}
               controlRef={controlRef}
               validationKey={key}
+              collapsed={collapsed}
```

Comment:
Quick question (even after this has been merged).

Isn't the hiding of object controls already handled a few lines above this through the `display: none` of the `collapsedObjectControl` style?

https://github.com/netlify/netlify-cms/blob/a9d0699a82a25d79dd99af4f1ed9643ce7bba220/packages/netlify-cms-widget-list/src/ListControl.js#L396-L400